### ISSUE TITLE
[FEATURE] IPv6 support

### ISSUE THREAD

**Issue by freeacetone** [hashtopolis/server#725]
Latest release `0.12.0` (and early) not support work via IPv6 connection. For example, start page says:
```
To add new agents provide them with a valid voucher and download a client.
Used vouchers are automatically deleted to prevent double spending.
If you are asked to provide the API url on the client to connect to, you need to enter the following:
http://[201/api/server.php
```
`http://[201/api/server.php` instead `http://[201:a373:f940:2421:b2f2:6a91:86b7:5628]/api/server.php`

Also install procedure via IPv6 connection has everytime stopped at connection to db step. Via localhost works correctly.

Thanks.

**Comment by TheWorkingDeveloper** [comment]
On topic of this, agent IP also doesn't have enough characters in the MySQL field to support the IPv6 address

**Comment by acetoneRu** [comment]
So wrong. Fix possible?

**Comment by gluafamichl** [comment]
Fixed with PR #1076
